WcaOnRails/app/views/registrations/edit.html.erb

```diff
@@ -30,20 +30,39 @@
         <%= f.submit class: "btn btn-default" %>
       </div>
     </div>
-    <hr />
+  <% end %>
+  <hr />
+  <div>
+    <h4>Payment History</h4>
```

Comment:
Hmm... would it make sense to only show this if the competition actually has payments turned on?